src/List.js

```diff
@@ -182,34 +185,58 @@ export function List() {
       );
     }
   };
-
-  return listIsShown || items.length ? (
-    <>
-      <AddForm />
+  return items.length ? (
```

Comment:
this line needs to be, that is why there is a bug.

`  return listIsShown || items.length ? (`

Reply:
Yes, that resolves the bug, We should keep that page? because now all are on one page, that is the List page, I think it doesn't have a sense. I think that message should be is in the list page under the form when the item list have empty

Reply:
Yes, that resolves the bug, We should keep that page? because now all are on one page, that is the List page, I think it doesn't have a sense. I think that message should be is in the list page under the form when the item list has empty. @bajancode what do you think?

Reply:
Thanks for asking! I'll see if I can pull it to take a look if I can get away from my kiddo but since you've already merged I'll assume you made a good choice already!!